.travis-before-script.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+
+set -e $DRUPAL_TI_DEBUG
+
+# Ensure the right Drupal version is installed.
+# Note: This function is re-entrant.
+drupal_ti_ensure_drupal
+
+# Add needed dependencies.
+cd "$DRUPAL_TI_DRUPAL_DIR"
+
+# These variables come from environments/drupal-*.sh
+mkdir -p "$DRUPAL_TI_MODULES_PATH"
+cd "$DRUPAL_TI_MODULES_PATH"
+
+#Enable php-redis
+sudo echo "extension = redis.so" >> ~/.phpenv/versions/$(phpenv version-name)/etc/php.ini
```

Comment:
Is this the only reason we need sudo? @LionsAd, is there a sudo-less way to do this?